crates/bevy_internal/Cargo.toml

```diff
@@ -197,7 +197,7 @@ bevy_state = { path = "../bevy_state", optional = true, version = "0.14.0-dev" }
 bevy_hierarchy = { path = "../bevy_hierarchy", version = "0.14.0-dev" }
 bevy_input = { path = "../bevy_input", version = "0.14.0-dev" }
 bevy_log = { path = "../bevy_log", version = "0.14.0-dev" }
-bevy_math = { path = "../bevy_math", version = "0.14.0-dev" }
+bevy_math = { path = "../bevy_math", version = "0.14.0-dev", features = ["reflect"] }
```

Comment:
I’m not sure we want to enable the feature here. I don't think any other package with an option `bevy_reflect` dependency does this.

Reply:
Ok, will change it after I fix the compilation errors.